services/order.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, MovieSession, Ticket
+
+
+def create_order(tickets: list, username: str, date: str = None) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user_id=user.id)
+        if date:
+            order.created_at = date
+            order.save()
+        for ticket in tickets:
+            session = MovieSession.objects.get(id=ticket["movie_session"])
+            Ticket.objects.create(
+                movie_session=session,

```

Comment:
can be replaced with 
```python
Ticket.objects.create(movie_session_id=ticket["movie_session"],...
```

Reply:
that was on the checklist for the previous task
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md